### PR TITLE
Simulate exponential weights

### PR DESCRIPTION
**Pull request by s-ben**
Not looking to merge this PR, just thought this would be a good place to have convo. Can move elsewhere if this is problematic for Cred purposes. 

So, Maker is looking to experiment with the new exponential weighting feature we just merged into [v0.7.2](https://github.com/sourcecred/sourcecred/releases/tag/v0.7.2). To help them evaluate whether this makes sense (or if they should stick with our current balanced/immediate policy for now), I want to calculate what the payouts would be using exponential weighting for the last ~3 weeks. NOTE: this date range was largely chosen for convenience, and because it omits some large Cred holders that opted in last week, skewing any comparison; my method for calculating new ratios using the existing policy is to just revert to this [commit](https://github.com/sourcecred/makerdao-cred/commit/9d8513e732f3dd9b888cc9378af6791368f11648) and rerun scores with a new ratio. My current plan for calculating the exponential weighting is to re-run from the same commit, but with exponential weighting.  

A couple questions:
- Since this option isn't available in the `grain.json` config file yet, is it possible for me to hard-code this? Would it be easier for someone to just do this for me on this PR? 
- I attempted to upgrade the instance to v0.7.2 in the PR, is this the correct way to do it? First time doing that...
- What exponential weighting (discount factor) should I use here? What ratio would most closely match payouts using the existing 75/20 balanced/immediate policy? 50/50 (another ratio we're looking at)?

### CODE REVIEW

**Comment by elihanover** [comment]
@s-ben I'm not sure how to answer these off the top of my head, but just to make you aware, my vision for these grain allocation modules is to build a frontend explorer that lets you map out distributions of policies and combinations of policies with different params.  It's not in the works quite yet, but I just wanted to let you know.

**Comment by elihanover** [comment]
Also it the config should be setup now so you can try it with different discounts.  I'm not sure what discount to use, as I haven't modeled out the distributions of the other policies yet.

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "template-instance",
-  "version": "0.1.0",
+  "version": "0.7.2",
```
Comment (decentralion) [inline package.json:3]: This is the template-instance version, which is independent from the sourcecred version. No reason to change it.

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   "description": "A template repo for creating a SourceCred instance",
   "repository": "git@github.com:sourcecred/template-instance.git",
   "author": "SourceCred Team <contact@sourcecred.io>",
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "sourcecred": "0.7.1"
+    "sourcecred": "0.7.2"
```
Comment (decentralion) [inline package.json:10]: Once 0.7.4 gets released (tonight afaik) you'll be able to actually set a distribution policy that uses the exponential policy (as @elihanover said)

Comment (s-ben) [reply package.json:10]: Sweet. Will run it hopefully tomorrow. Do you or @elihanover have a suggested discount to try with Maker? Something that perhaps is close'ish to what they have now (75/25 balanced/immediate)? Or perhaps 50/50 balanced/immediate, which is another option they're considering?

**Pull request closed without merging**